packages/data-provider/src/utils/submissionUtils.ts

```diff
@@ -25,12 +25,18 @@ import {
 	MERGE_REFERENCE_TYPE,
 	SUBMISSION_STATUS,
 	SubmissionReference,
+	SubmissionStatus,
 	SubmittedDataReference,
 } from './types.js';
 
 const utils = (dependencies: BaseDependencies) => {
 	const LOG_MODULE = 'SUBMISSION_UTILS';
 	const { logger } = dependencies;
+
+	// Only "open", "valid", and "invalid" statuses are considered Active Submission
+	const statusesAllowedToClose = [SUBMISSION_STATUS.OPEN, SUBMISSION_STATUS.VALID, SUBMISSION_STATUS.INVALID];
```

Comment:
I think this has to be `as const` for the type inference on the next line to read the literal values instead of just type `string`

```ts
const statusesAllowedToClose = [SUBMISSION_STATUS.OPEN, SUBMISSION_STATUS.VALID, SUBMISSION_STATUS.INVALID] as const;
```

Reply:
changed as suggested. thanks for noticing